features/environment.py

```diff
@@ -14,7 +14,7 @@ def before_feature(context, feature):
     # between features so we need to preserve only the bottom two stack
     # frames when beginning a new feature.
     context._stack = context._stack[-2:]
-    
+    context.error_on_passed_rule = '--tags=-error_on_passed_rule' in sys.argv
```

Comment:
I think we should use `--input` for this rather than `--tags`.

Reply:
I am not sure if I get that 100%. As I understood it, `input` is the file path, so something that can be exactly the same, whether we use Gherkin as a script or as a test_main.py / validation repo. How do we recognize which instance invoked the script by `input`?

The --tags was an idea since it uses the negative tag functionality. That way it's "invisible" in the feature files, feature file devs don't need to think about it, but nonetheless, it explicitly defines the expected behavior.

Reply:
Sorry I meant `--define` for passing arbitrary variables to behave rather than tags which have a specific meaning already. You're now relying on the fact that we don't (ok, and will not have) error_on_passed_rule as a regular tag. But using --define would be more pure I'd say.

We use --define for specifying `input` [0] hence my confusing. Thanks for asking.

[0] https://github.com/buildingSMART/ifc-gherkin-rules/blob/main/main.py#L62